app/src/common/urls.js

```diff
@@ -93,6 +93,17 @@ export const URLS = {
     `${urlBase}${activeProject}/launch/attribute/values?${
       key ? `filter.eq.attributeKey=${key}&` : ''
     }filter.cnt.attributeValue=${searchTerm}`,
+  itemAttributeKeysByLaunchName: (activeProject, launchName) => (searchTerm = '') =>
+    `${urlBase}${activeProject}/item/step/attribute/keys${getQueryParams({
+      'filter.eq.name': launchName || undefined,
+      'filter.cnt.attributeKey': searchTerm,
+    })}`,
+  itemAttributeValuesByLaunchName: (activeProject, launchName, key) => (searchTerm = '') =>
+    `${urlBase}${activeProject}/item/step/attribute/values${getQueryParams({
+      'filter.eq.name': launchName || undefined,
```

Comment:
Can these properties differ from `undefined` or `some_string_value`?

Reply:
No, it can't